chain-tx-enclave-next/enclave-ra/ra-client/test/Intel_SGX_Attestation_RootCA.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I guess this one is the same as https://github.com/crypto-com/chain/blob/master/client-common/src/cipher/AttestationReportSigningCACert.pem -- no need to duplicate it?